### PR TITLE
Concurrency testing

### PR DESCRIPTION
**Pull request by finn-block**
this is some code to make some concurrent HTTP requests to make sure that works. Most of it probably doesn't need to get merged.

### CODE REVIEW

File: impl/concurrencytest/main.go [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+					log = log.WithError(err)
+				}
+				log.WithField("time", time.Since(putStart)).Info("PUT request completed")
+				if err != nil {
+					return
+				}
+
+				getStart := time.Now()
+				err = get(s, suffix)
+				if err != nil {
```
Comment (thereisnogabe) [inline impl/concurrencytest/main.go]:
```suggestion
				if err = get(s, suffix); err != nil {
```

**Review by thereisnogabe — APPROVED** [review]
excellent!

**Comment by thereisnogabe** [comment]
@finn-tbd your call what needs to get merged or not. It would be good to run this post-merge is my guess. every PR may be overkill. your call

**Comment by finn-block** [comment]
i can't see a lot of value to adding it to CI. it will slow down the pipeline pretty bad. I'll merge it so we can have it available if we have a specific reason to validate future changes against something like this. Maybe if it becomes an ongoing issue, we can add it to the CI.

**Pull request merged**